### PR TITLE
modification for torch lightning compatibility

### PR DESCRIPTION
**Pull request by hyunwoongko**
### Issues
- [#7] (torch-lightning에서 발생하는 GPU 문제)

### Tasks
- [X] GPT2와 torch lightning간의 호환성을 위한 작업
- [X] Meena와 torch lightning간의 호환성을 위한 작업

### Changes
- 클래스 내에서 생성되는 텐서의 gpu allocation 방식을 변경했습니다.
- 기존 : torch.arrange(... , device=device)
- 변경 : torch.arrange(...).to(device)

### CODE REVIEW

**Comment by hyunwoongko** [comment]
이렇게 해도 몇몇 버그가 사라지지 않네요 음...

**Comment by hyunwoongko** [comment]
torch-lightning의 gradient clipping 버그인 것 같습니다. 
다른 모델들 학습할땐 괜찮은데 올려주신 Meena 코드로 학습하면서 clipping을 켜면 에러가 발생하네요. 
일단 clipping 끄니까 다행히 정상 학습 가능했습니다.

**Comment by harrydrippin** [comment]
안녕하세요! 이미 #7 에서 이슈를 해결하신 듯 하지만, 추가로 설명드리겠습니다. 현재는 저희 프로젝트가 PyTorch Lightning에 호환되도록 작성되어 있지 않아서 관련된 고려가 빠져있습니다. 만약 이 이슈와 관련한 패치를 보유하고 계신다면 부담 없이 PR을 넣어주셔도 좋을 것 같습니다 👍
